parser/src/main/kotlin/com/guerinet/mymartlet/model/Course.kt

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2014-2018 Julien Guerinet
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.guerinet.mymartlet.model
+
+import java.time.DayOfWeek
```

Comment:
You're going to need to use the Backport here, as the Java 8 Date API isn't available on every Android version (yet).

Reply:
I'll add it later today. As a note, [threetenbp](https://github.com/ThreeTen/threetenbp) is added through [ThreeTenABP](https://github.com/JakeWharton/ThreeTenABP), whereas there will be no android dependencies in this module. I will add the latest version, but once parser gets used in app, the version may need to be downgraded to match. You can probably ignore the package from the import if the API hasn't changed.